packages/core/src/components/forms/form-group.md

```diff
@@ -4,6 +4,17 @@ Form groups support more complex form controls than [simple labels](#core/compon
 such as [control groups](#core/components/forms/control-group) or [`NumericInput`](#core/components/forms/numeric-input).
 They also support additional helper text to aid with user navigation.
 
+@reactExample FormGroupExample
+
+@## JavaScript API
```

Comment:
we're still in the habit of CSS API appearing before JS API so there's no good reason to move only this one.